projects/storefrontlib/src/cms-components/user/register/register.component.ts

```diff
@@ -20,8 +20,7 @@ import {
 } from '@spartacus/core';
 import { combineLatest, Observable, Subscription } from 'rxjs';
 import { filter, map, tap } from 'rxjs/operators';
-import { sortTitles } from '../../../shared/utils/forms/title-utils';
-import { CustomFormValidators } from '../../../shared/utils/validators/custom-form-validators';
+import { sortTitles, CustomFormValidators } from '../../../shared/index';
 
 @Component({
```

Comment:
Document breaking changes:
- renamed submit to submitForm

Also worth mentioning that there is a new method register

Reply:
Done